homework/inner-product/arithmeticAverage.cpp

```diff
@@ -0,0 +1,31 @@
+#include "arithmeticAverage.hpp"
+#include <numeric>
+#include <cmath>
+
+float ArithmeticAverage(const std::vector<int>& v1, const std::vector<int>& v2){
+    float sum = 0.f;
+    float length = v1.size() + v2.size();
+    
+    for(const auto el : v1){
+        sum += el;
+    }
+    
+    for(const auto el : v2){
+        sum += el;
+    }
+    
+    return sum/length;
```

Comment:
As the name of homework said :) INNER_PRODUCT

so try to solve how you can use inner product here. 

Tip: you can add 2 vecotrs using inner product, try to also think about how to divide values